catroid/src/main/java/org/catrobat/catroid/content/bricks/SetLookBrick.java

```diff
@@ -231,7 +231,7 @@ public Brick clone() {
 
 	@Override
 	public List<SequenceAction> addActionToSequence(Sprite sprite, SequenceAction sequence) {
-		sequence.addAction(sprite.getActionFactory().createSetLookAction(getSprite(), look, wait));
+		sequence.addAction(sprite.getActionFactory().createSetLookAction(sprite, look, wait));
```

Comment:
Checking something is always a great idea. :smirk_cat:

Since you commented on this line. The difference between `getSprite` and `sprite` is
* `getSprite` returns the current opened sprite in the IDE. There is always only _one_ current sprite. It's just a helper method. `getCurrentSprite` may be a better name because it's similar to `ProjectManager.getInstance().getCurrentSprite()`. Or somehow mention that's the current Sprite of the Ide :wink:.
* `sprite` is the Sprite where the brick is in.

The method [SetBackgroundBrick.java#L36](https://github.com/Catrobat/Catroid/blob/b5a9fc8323373031b4d561cba3d69de77d094319/catroid/src/main/java/org/catrobat/catroid/content/bricks/SetBackgroundBrick.java#L36) violates this. If there is only a single Scene, there won't be a problem. However, if there are multiple Scenes (`ProjectManager.getInstance().__getCurrentScene()__.getSpriteList().get(0);`), and the method is misused, it's harder to detect the error. I would suggest not to override it.